DS-Session4-Question1/src/meta/test/Main.java

```diff
@@ -0,0 +1,42 @@
+package meta.test;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+
+import meta.utility.Employee;
+import meta.utility.NaturalSort;
+import meta.utility.SortByName;
+import meta.utility.UniqueEmployee;
+
+public class Main {
```

Comment:
As per my understanding, you can have 3 different main functions/programs to suffice the need of the question. There will be one Employee class, and 2 classes implementing comparable (natural sort, sort by name).